kb/prometheus-grafana-setup-cloud.md

```diff
@@ -0,0 +1,259 @@
+When you run Prometheus locally, set your target address to port 8077 in your Prometheus configuration YAML file. (We set the scrape endopint to port 8077 in the [SDK metrics setup](#sdk-metrics-setup) example.)
+
+Example:
+
+```
+global:
+ scrape_interval: 10s # Set the scrape interval to every 10 seconds. Default is every 1 minute.
+#...
+
+# Set your scrape configuration targets to the ports exposed on your endpoints in the SDK.
+scrape_configs:
+ - job_name: 'temporalsdkmetrics'
+   metrics_path: /metrics
+   scheme: http
+   static_configs:
+     - targets:
+       # This is the scrape endpoint where Prometheus listens for SDK metrics.
+       - localhost:8077
+       # You can have multiple targets here, provided they are set up in your application code.
+```
+
+See the Prometheus documentation for more details on how you can run Prometheus locally or using Docker.
+
+Note that Temporal Cloud exposes metrics through a Prometheus HTTP API endpoint (not a scrape endpoint) that can be configured as a data source in Grafana.
+The Prometheus configuration described here is for scraping metrics data on endpoints for SDK metrics only.
+
+To check whether Prometheus is receiving metrics from your SDK target, go to [http://localhost:9090](http://localhost:9090) and navigate to **Status&nbsp;> Targets**.
+The status of your target endpoint defined in your configuration appears here.
+
+## Data sources configuration for Temporal Cloud and SDK metrics in Grafana
+
+Depending on how you use Grafana, you can either install and run it locally, run it as a Docker container, or log in to Grafana Cloud to set up your data sources.
+
+If you have installed and are running Grafana locally, go to [http://localhost:3000](http://localhost:3000) and sign in.
+
+You must configure your Temporal Cloud and SDK metrics data sources separately in Grafana.
+
+To add the Temporal Cloud Prometheus HTTP API endpoint that we generated in the [Temporal Cloud metrics setup](#temporal-cloud-metrics-setup) section, do the following:
+
+1. Go to **Configuration&nbsp;> Data sources**.
+2. Select **Add data source&nbsp;> Prometheus**.
+3. Enter a name for your Temporal Cloud metrics data source, such as _Temporal Cloud metrics_.
+4. In the **HTTP section**, paste the URL that was generated in the Observability section on the Temporal Cloud UI.
+5. In the **Auth section**, enable **TLS Client Auth**.
+6. In the **TLS/SSL Auth Details** section, paste the end-entity certificate and key. Note that the end-entity certificate used here must chain with the root CA certificates used in your [Temporal Cloud observability setup](#temporal-cloud-metrics-setup).
+7. Select **Save and test** to ensure the data source is working properly.
+
+If you see issues in setting this data source, verify that your end-entity and root CA certificates are chained, and you are setting the correct certificates in your Temporal Cloud observability setup and in the TLS authentication in Grafana.
```

Comment:
### 
```suggestion
If you see issues in setting this data source, verify your CA certificate chain, and verify you are setting the correct certificates in your Temporal Cloud observability setup and in the TLS authentication in Grafana. You can verify your certificates using the following command:
```
$ openssl s_client -connect <namespace>.<accountId>.tmprl.cloud:7233 -showcerts -cert /path/to/ca.pem -key /path/to/ca.key -tls1_2 -servername <namespace>.<accountId>.tmprl.cloud
```
```

Reply:
`$ openssl s_client -connect <namespace>.<accountId>.tmprl.cloud:7233 -showcerts -cert /path/to/ca.pem -key /path/to/ca.key -tls1_2 -servername <namespace>.<accountId>.tmprl.cloud`

@ktenzer I think this command tests if the certificates work with the <namespace>, not with the observability endpoint.